docker/minimal/core-docker-compose.yml

```diff
@@ -3,7 +3,7 @@ version: '3'
 services:
   core0:
     container_name: core0
-    image: "celestia-app:v0.7.0"
+    image: "ghcr.io/celestiaorg/celestia-app:v0.10.0-rc1"
```

Comment:
Unfortunately, if we are bumping to v0.10.0-rc1 than we would also need to bump node images to v0.5.0-rc5 🙏

Reply:
Updated to 0.5.0-rc5